src/Dynamo.All.sln

```diff
@@ -1,17 +1,26 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 17
-VisualStudioVersion = 17.0.32014.148
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.32106.194
```

Comment:
changed back the vs version as before 
(It gets changed automatically based on what version of VS you used when saving the sln file)